hw/ip/otp_ctrl/data/otp_ctrl.hjson

```diff
@@ -947,6 +950,9 @@
       swaccess: "rw",
       hwaccess: "hro",
       regwen:   "CHECK_REGWEN",
+      tags:     [ // Triggering any of these checks may lead to unintended alerts
+                  // if the CHECK_TIMEOUT is not configured properly.
+                  "excl:CsrAllTests:CsrExclWrite"],
```

Comment:
not sure if it's better to exclude the check for alert related registers, so that we can still verify if this reg is locked correctly based on regwen `CHECK_REGWEN` in our common csr seq. same for the other regs which lead to alerts.
@cindychip

Reply:
that is a good point, thanks for tagging me. If the exclusions in this PR https://github.com/lowRISC/opentitan/pull/3905 are applied, I do not think we need this one right Michael?

Reply:
(Note that this change is part of a separate PR that I just merged in, so we would have to make the changes upstream.)

@weicaiyang are you recommending we do not add the `"excl:CsrAllTests:CsrExclWrite"` tag here? I believe the issue is that this has unintended side effects in other CSRs (like the error code and status regs) if we do not add this tag here... @cindychip can you maybe double check whether we really need this exclusion tag, now that we've fixed other issues in the chip-level tests?

Reply:
sorry I think we commented simultaneously. the CSR exclusions that @weicaiyang commented on where part of #3905, actually (this PR here was dependent on #3905). 

but we may have been overly pessimistic while adding them, since the main issue leading to timeouts in #3905 was actually due to the uninitialized OTP memory array. so we could consider removing them...

Reply:
I feel we could remove "excl:CsrAllTests:CsrExclWrite" here and exclude the check for error code and status regs as automatic CSR seq can't know when the values of these regs will be changed and we have to test those separately , but CSR seq can test if locked reg is write-able.

Reply:
sounds good, I've done that here: https://github.com/lowRISC/opentitan/pull/3937